README.md

```diff
@@ -37,18 +32,13 @@ The environment variables are:
 In order to mock your environment so you can **test** things, you can have a
 script called `env.sh` for example with the contents of:
 ```
-export OPENDAP_DIR='/tmp/opendap'
-export PUBLIC_DIR='/tmp/public'
 export ARCHIVE_DIR='/tmp/archive'
 export INCOMING_DIR='/tmp/incoming'
-export OPENDAP_IMOS_DIR="$OPENDAP_DIR/1/IMOS/opendap"
-export PUBLIC_IMOS_DIR="$PUBLIC_DIR"
-export ARCHIVE_IMOS_DIR="$ARCHIVE_DIR"
 export WIP_DIR='/tmp/wip'
 export DATA_SERVICES_DIR="$PWD"
 export LOG_DIR='/tmp/log'
 
-mkdir -p $OPENDAP_IMOS_DIR $PUBLIC_IMOS_DIR $ARCHIVE_IMOS_DIR $WIP_DIR $LOG_DIR
+mkdir -p $ARCHIVE_DIR $WIP_DIR $LOG_DIR
```

Comment:
Do we still need this? Testing now happens in the VM, where the env is set up with chef.
If we do want to leave it, should also include `mkdir  $INCOMING_DIR`